.github/workflows/build-terraform-cli.yml

```diff
@@ -51,24 +51,29 @@ jobs:
           GO_LDFLAGS: ${{ inputs.ld-flags }}
           ACTIONSOS: ${{ inputs.runson }}
           CGO_ENABLED: ${{ inputs.cgo-enabled }}
-        uses: hashicorp/actions-go-build@v0.1.7
+        uses: hashicorp/actions-go-build@e20c6be7bf010e40e930dab20e6da63176725ec1 # v0.1.9
```

Comment:
I noticed that we were a couple of versions behind of this action so it seemed like a good idea to upgrade, but I do worry a little whether I'm introducing more change in this PR than I need to.

Reply:
this should be totally safe to do - other product repos are using[ 0.1.9 ](https://github.com/hashicorp/boundary/blob/8e2ac23a65f91acf679878fe800e0e686eb93aa8/.github/workflows/build.yml#L178 )successfully